File: Iris/Iris.Twitter/Tweet.cs
using System;
using System.Text.Json.Serialization;

namespace Iris.Twitter
{
    internal class Tweet
    {
        [JsonPropertyName("entries")]
        public Media Media { get; set; }
        
        [JsonPropertyName("isRetweet")]
        public bool IsRetweet { get; set; }

        [JsonPropertyName("likes")]
        public int Likes { get; set; }

        [JsonPropertyName("replies")]
        public int Replies { get; set; }

        [JsonPropertyName("retweets")]
        public int Retweets { get; set; }
        
        [JsonPropertyName("text")]
        public string Text { get; set; }

        [JsonPropertyName("time")]
        [JsonConverter(typeof(DateTimeConverter))]
        public DateTime Date { get; set; }
        
        [JsonPropertyName("tweetId")]
        [JsonConverter(typeof(StringToLongConverter))]
        public long Id { get; set; }
    }
}

File: twitterproducer/tweet.py
from datetime import datetime


class Tweet:
    tweetId: str
    userId: str
    username: str
    tweetUrl: str
    isRetweet: bool
    isPinned: bool
    time: datetime
    text: str
    replies: int
    retweets: int
    likes: int
    entries: dict

    def __init__(self, original_dict):
        self.__dict__ = original_dict
